package.json

```diff
@@ -69,6 +69,7 @@
     "@types/node": "13.1.8",
     "@types/sinon": "9.0.5",
     "@types/ws": "7.2.0",
+    "@types/mime": "2.0.3",
```

Comment:
From what I see `@types/mime` is not used directly, but used by some other library. Perhaps update that library instead otherwise it could be a bit confusing where is this library used.

Reply:
Sounds good, it's from express, I'll find the update that it works with and make sure it has no breaking changes!

Reply:
Hmm, actually no update of express helps here. The dependency tree goes like this:
`"@types/mime": "*"` <- `"@types/serve-static": "*"` <- `@types/express` <- `"@types/express": "^4.17.2"` <- `twilio`

Reply:
I could also update `@types/serve-static`, but that wouldn't be much different either

Reply:
I tried putting it to "overrides" section of `package.json`, but this did not have any effect on the outcome, as it did not put the correct version after installing the packages.

Reply:
@ostap0207 maybe let's merge it like this for now and in the PR to Twilio master, perhaps they have better ideas on what to do with this package situation. It's currently unrelated to the main issue anyway and it'll probably be fixed soon.